<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetControlPanel</class>
 <widget class="QWidget" name="WidgetControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;
color:white;
font-family:&quot;맑은 고딕&quot;;</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>251</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Translation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>50</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Increment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>88</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#494949;
}</string>
   </property>
   <property name="text">
    <string>Object Opacity</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_11">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>50</y>
     <width>23</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 #B4B4B4, stop:1 #494949);
}

QToolButton:hover {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0.324, y1:0.306682, x2:1, y2:1, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	color:white;
}</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_18">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	border:none;
	background-color: #494949;
	color:white;
}

QToolButton:hover {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0.153409, y1:0.147727, x2:0.664773, y2:0.682, stop:0 rgba(255, 187, 147, 253), stop:1 rgba(255, 143, 133, 255));
}</string>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_14">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>151</y>
     <width>23</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 #B4B4B4, stop:1 #494949);
}

QToolButton:hover {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0.324, y1:0.306682, x2:1, y2:1, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	color:white;
}</string>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>151</y>
     <width>55</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_13">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>28</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color: #494949;
	border:none;
	color:white;
	position:absolute;
}

QToolButton:hover {
	border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #494949, stop:1 #FF886D);
	color:white;
}
</string>
   </property>
   <property name="text">
    <string>—</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>84</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal {
    border: 1px solid #494949;
	height:10px;
    width: 15px;
    margin: -8px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 8px;
	background-color:#494949;
	padding:-10px 0px;
}
QSlider::groove:horizontal {
	border-radius:1px;
	border:none;
	height:5px;
	margin:0px;
    color:#494949;
}

QSlider::sub-page
{
    background: #494949;
}

QSlider::add-page 
{
    background: rgb(70, 70, 70);
}</string>
   </property>
   <property name="sliderPosition">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_19">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	border:none;
	background-color: #494949;
	color:white;
}

QToolButton:hover {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0.153409, y1:0.147727, x2:0.664773, y2:0.682, stop:0 rgba(255, 187, 147, 253), stop:1 rgba(255, 143, 133, 255));
}</string>
   </property>
   <property name="text">
    <string>Translation</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>188</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
color:white;
border:1px solid #494949;
background-color:black;
}

QDoubleSpinBox::up-button {
	width:30px;
}

QDoubleSpinBox::down-button {
	width:30px
}</string>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>288</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
color:white;
border:1px solid #494949;
background-color:black;
}

QDoubleSpinBox::up-button {
	width:30px;
}

QDoubleSpinBox::down-button {
	width:30px
}</string>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_17">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	border:none;
	background-color: #494949;
	color:white;
}

QToolButton:hover {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0.153409, y1:0.147727, x2:0.664773, y2:0.682, stop:0 rgba(255, 187, 147, 253), stop:1 rgba(255, 143, 133, 255));
}</string>
   </property>
   <property name="text">
    <string>Boundary</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_16">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>53</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	border:none;
	background-color: #494949;
	color:white;
}

QToolButton:hover {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0.153409, y1:0.147727, x2:0.664773, y2:0.682, stop:0 rgba(255, 187, 147, 253), stop:1 rgba(255, 143, 133, 255));
}</string>
   </property>
   <property name="text">
    <string>Object</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_15">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>251</y>
     <width>23</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 #B4B4B4, stop:1 #494949);
}

QToolButton:hover {
	border:none;
	background-color:qlineargradient(spread:pad, x1:0.324, y1:0.306682, x2:1, y2:1, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	color:white;
}</string>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_12">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color: #494949;
	border:none;
	color:white;
	position:absolute;
}

QToolButton:hover {
	border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #FF886D, stop:1 #494949);
	color:white;
}
</string>
   </property>
   <property name="text">
    <string>┼</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>28</y>
     <width>106</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#FFD3A7;
}</string>
   </property>
   <property name="text">
    <string>Boundary Opacity</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal {
    border: 1px solid #494949;
	height:10px;
    width: 15px;
    margin: -8px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 8px;
	background-color:#FFD3A7;
	padding:-10px 0px;
}
QSlider::groove:horizontal {
	border-radius:1px;
	border:none;
	height:5px;
	margin:0px;
    color:#FFD3A7;
}

QSlider::sub-page
{
    background: #FFD3A7;
}

QSlider::add-page 
{
    background: rgb(70, 70, 70);
}</string>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>toolButton_11</zorder>
  <zorder>toolButton_14</zorder>
  <zorder>label_7</zorder>
  <zorder>toolButton_13</zorder>
  <zorder>Slider</zorder>
  <zorder>doubleSpinBox</zorder>
  <zorder>doubleSpinBox_2</zorder>
  <zorder>toolButton_17</zorder>
  <zorder>toolButton_16</zorder>
  <zorder>toolButton_15</zorder>
  <zorder>toolButton_12</zorder>
  <zorder>label_9</zorder>
  <zorder>Slider_2</zorder>
  <zorder>toolButton_18</zorder>
  <zorder>toolButton_19</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
